rascaline/src/calculators/lode/spherical_expansion.rs

```diff
@@ -353,30 +547,40 @@ impl CalculatorBase for LodeSphericalExpansion {
 
 #[cfg(test)]
 mod tests {
+    use crate::systems::test_utils::test_system;
+    use crate::Calculator;
+    use crate::calculators::CalculatorBase;
+
     use super::*;
-    use ndarray::arr3;
+
+    fn parameters() -> LodeSphericalExpansionParameters {
+        LodeSphericalExpansionParameters {
+            cutoff: 3.5,
+            max_radial: 6,
+            max_angular: 6,
+            atomic_gaussian_width: 0.8,
+            potential_exponent: 1,
+            radial_basis: LodeRadialBasis::splined_gto(1e-8),
+        }
+    }
 
     #[test]
-    fn test_compute_structure_factors() {
-        let k_vectors = [
-            KVector{direction: Vector3D::new(1.0, 0.0, 0.0), norm: 1.0},
-            KVector{direction: Vector3D::new(0.0, 1.0, 0.0), norm: 1.0},
-        ];
-
-        let positions = [Vector3D::new(1.0, 1.0, 1.0), Vector3D::new(2.0, 2.0, 2.0)];
-
-        let structure_factors = compute_structure_factors(&positions, &k_vectors);
-
-        let expected_real= arr3(&[
-            [[2.0, 2.0], [1.0806046117362793, 1.0806046117362793]],
-            [[1.0806046117362793, 1.0806046117362793], [2.0, 2.0]]
-        ]);
-        let expected_imag = arr3(&[
-            [[0.0, 0.0], [-1.682941969615793, -1.682941969615793]],
-            [[1.682941969615793, 1.682941969615793], [0.0, 0.0]]
-        ]);
-
-        assert_eq!(structure_factors.real, expected_real);
-        assert_eq!(structure_factors.imag, expected_imag);
```

Comment:
Is there a specific reason to remove the tests here?

Reply:
This is temporary, I should add them back in. I changed the structure factor layout a bit, removing the part which was tested here. We now store `exp(i k ri)` and `\sum_j exp(i k (rj  - ri))`, and no longer the full `exp(i k (rj  - ri))`

Reply:
Okay.